exonum-java-binding-fakes/src/test/java/com/exonum/binding/fakes/mocks/ThrowingTransactionsTest.java

```diff
@@ -35,7 +35,7 @@ void createThrowingIllegalArgument() {
     Class<IllegalArgumentException> exceptionType = IllegalArgumentException.class;
     Transaction transaction = ThrowingTransactions.createThrowing(exceptionType);
 
-    assertThrows(exceptionType, transaction::isValid);
+    assertThrows(exceptionType, () -> transaction.execute(null));
```

Comment:
Can we pass a proper context? `null` does not seem like an acceptable one, distracting from the test